src/sth_helper.js

```diff
@@ -166,6 +156,39 @@
       pad(date.getUTCMilliseconds(), true) + 'Z';
   }
 
+  /**
+   * Transforms a response payload into a NGSI formatted response
+   *  payload
+   * @param entityId The id of the requested entity's data
+   * @param attributeId The id of the requestedattribute's data
+   * @param payload The payload to transform
+   * @return {Object} The payload using NGSI format
+   */
+  function getNGSIPayload(entityId, attributeId, payload) {
+    var ngsiResponse = {
```

Comment:
I love you don't have to create such a string var by using quotes and string concatenations. That's something really ugly about Java.

NTC